draft-zf-community-dev-fund-2-proposal.rst

```diff
@@ -0,0 +1,486 @@
+
+
+Requirements
+============
+
+The Dev Fund should encourage decentralization of the work and funding, by 
+supporting new teams dedicated to Zcash.
+
+The Dev Fund should maintain the existing teams and capabilities in the Zcash 
+ecosystem, unless and until concrete opportunities arise to create even 
+greater value for the Zcash ecosystem.
+
+There should not be any single entity which is a single point of failure, 
+i.e., whose capture or failure will effectively prevent effective use of the 
+funds.
+
+Major funding decisions should be based, to the extent feasible, on inputs 
+from domain experts and pertinent stakeholders.
+
+The Dev Fund mechanism itself should not modify the monetary emission curve 
+(and in particular, should not irrevocably burn coins).
+
+In case the value of ZEC jumps, the Dev Fund recipients should not wastefully 
+use excessive amounts of funds. Conversely, given market volatility and 
+eventual halvings, it is desirable to create rainy-day reserves.
+
+The Dev Fund mechanism should not reduce users' financial privacy or security. 
+In particular, it should not cause them to expose their coin holdings, nor 
+cause them to maintain access to secret keys for much longer than they would 
+otherwise. (This rules out some forms of voting, and of disbursing coins to 
+past/future miners.)
+
+The new Dev Fund system should be simple to understand and realistic to 
+implement. In particular, it should not assume the creation of new mechanisms 
+(e.g., election systems) or entities (for governance or development) for its 
+execution; but it should strive to support and use these once they are built.
+
+The Dev Fund should comply with legal, regulatory, and taxation constraints in 
+pertinent jurisdictions.
+
+
+Non-requirements
+================
+
+General on-chain governance is outside the scope of this proposal.
+
+Rigorous voting mechanisms (whether coin-weighted, holding-time-weighted or 
+one-person-one-vote) are outside the scope of this proposal, though there is 
+prescribed room for integrating them once available.
+
+
+Specification
+=============
+
+Consensus changes implied by this specification are applicable to the Zcash 
+Mainnet. Similar (but not necessarily identical) consensus changes SHOULD be 
+applied to the Zcash Testnet for testing purposes.
+
+
+Dev Fund allocation
+-------------------
+
+Starting at the second Zcash halving in 2024, until block height 3146400 
+(which is expected to occur approximately one year after the second Zcash 
+halving), 20% of the block subsidy of each block SHALL be allocated to a "Dev 
+Fund" that consists of the following three slices:
+
+* 28% for the Bootstrap Project (denoted **BP slice**);
+* 32% for the Zcash Foundation (denoted **ZF slice**);
+* 40% for the Financial Privacy Foundation, for "Zcash Community Grants" for
+  large-scale long-term projects (denoted **ZCG slice**).
+
+Subsequently, starting at block height 3146400, 20% of the block subsidy of 
```

Comment:
This section is subject to change if Bootstrap/ECC declare that they are willing to accept funding for the full two years, instead of just one year.